src/components/pages/AllProjects.tsx

```diff
@@ -171,21 +190,27 @@ const AllProjects = () => {
                 defaultPageSize={25}
               />
             ) : (
-              <GenericTableWithData<
-                GetOrganizationsQuery,
-                GetOrganizationsQueryVariables,
-                OrganizationType
-              >
-                queryDocument={GetOrganizationsDocument}
-                columns={ORGANIZATION_COLUMNS}
-                rowLinkTo={organizationRowLink}
-                noData='No organizations'
-                pagePath='organizations'
-                showFilters
-                recordType='Organization'
-                defaultPageSize={25}
-                queryVariables={{}}
-              />
+              <Box>
+                <GenericTableWithData<
+                  GetOrganizationsQuery,
+                  GetOrganizationsQueryVariables,
+                  OrganizationType
+                >
+                  queryDocument={GetOrganizationsDocument}
+                  columns={ORGANIZATION_COLUMNS}
+                  rowLinkTo={organizationRowLink}
+                  noData='No organizations'
+                  pagePath='organizations'
+                  showFilters={false}
+                  defaultFilters={undefined}
+                  filters={undefined}
```

Comment:
These can be removed, since they are the defaults. It would be nice to have consistency with the Project table, but I agree it probably makes sense to just hide this since there are no available sort or filter options.

Reply:
Thanks 👍 I added these when I was trying to debug the issue with React thinking the 2 tables were the same component, and forgot to remove